docs/RtspStreamingGuide.md

```diff
@@ -0,0 +1,371 @@
+# USB Camera Device Service RTSP Streaming Guide
+
+## Contents
+
+[System Requirements](#system-requirements)  
+[How It Works](#how-it-works)  
+[Overview](#overview)  
+[Tested Devices](#tested-devices))  
+[Dependencies](#dependencies)  
+[Get the Source Code](#get-the-source-code)
+[Configuration options](#configuration-options)
+[Deploy the Service](#deploy-edgex-and-usb-device-camera-microservice)  
+[Verify the Service](#verify-service-and-device-profiles)   
+[Adding Devices using REST API](#adding-devices-using-rest-api)
+[Start Video Streaming](#start-video-streaming)
+[Shutting Down](#shutting-down)  
+[Troubleshooting](#troubleshooting)) 
+[License](#license)
+
+## System Requirements
+
+- Intel&#8482; Core&#174; processor
+- Ubuntu 20.04.4 LTS
+- USB-compliant Camera
+
+**Time to Complete**
+
+10-20 minutes
+
+**Other Requirements**
+
+You must have administrator (sudo) privileges to execute the user guide commands.
+
+## How It Works
+For an explanation of the architecture, see the [Readme](../README.md#how-it-works).
+
+## Overview
+EdgeX device service for communicating with USB cameras attached to Linux OS platforms.
+This service provides the following capabilities:
```

Comment:
I think that these need to follow the same form. Like Kimberly mentioned above, these should either all be complete sentences with descriptions of the capabilities or not.